go/systests/team_list_test.go

```diff
@@ -169,19 +169,32 @@ func TestTeamDuplicateUIDList(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	require.Equal(t, 2, len(details.Members.Writers))
-	for _, member := range details.Members.Writers {
-		require.Equal(t, bob.username, member.Username)
-		if member.Uv == uvBefore {
-			require.False(t, member.Active)
-			require.True(t, member.NeedsPUK)
-		} else if member.Uv == uvAfter {
-			require.True(t, member.Active)
-			require.False(t, member.NeedsPUK)
-		} else {
-			t.Fatalf("Unexpected UV for member: %v", member.Uv)
+	// Expecting just the active writer here, and not inactive
+	// (because of reset) invite.
+	require.Equal(t, 1, len(details.Members.Writers))
+	member := details.Members.Writers[0]
+	require.True(t, member.Active)
+	require.False(t, member.NeedsPUK)
+
+	_ = uvBefore
+	_ = uvAfter
+
+	// Commented out until we figure out the duplicate uvs story
```

Comment:
please make a ticket and delete this code; include the commented out code in the ticket.